packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -33,9 +33,41 @@
   overflow: hidden;
 }
```

Comment:
I would suggest using `display: inline-flex` on `.mdc-chip` to collapse whitespace in the markup. This will give you more control over spacing.

Then, instead of `vertical-align: middle` on the child elements, you can just set `align-items: center` on `.mdc-chip`.

Reply:
Done, thanks for the tip!